apps/web-client/src/routes/outbound/[id]/+page.svelte

```diff
@@ -619,7 +603,28 @@
 							await handleAddTransaction(isbn, 1, warehouseId);
 						}
 					}}
+					placeholder={"Scan to select books from"}
 				/>
+				<select
+					id="defaultWarehouse"
+					name="defaultWarehouse"
+					class="select-bordered select select-sm h-full"
+					value={defaultWarehouse || ""}
+					on:change={(e) => handleUpdateNoteWarehouse(parseInt(e.currentTarget.value))}
+				>
+					<option value="">{warehouses.length > 0 ? tOutbound.placeholder.any_warehouse() : tOutbound.placeholder.no_warehouses()}</option>
+					{#each warehouses as warehouse}
+						<option value={warehouse.id}>{warehouse.displayName}</option>
+					{/each}
+				</select>
+				<!-- <div id="button-container" class="flex h-24 w-full items-center justify-start px-8"> -->
+				<button
+					use:melt={$customItemDialogTrigger}
+					on:m-click={() => openCustomItemForm()}
+					on:m-keydown={() => openCustomItemForm()}
+					class="btn-neutral btn mx-2">{tOutbound.labels.custom_item()}</button
```

Comment:
I feel like a + icon before the text would be helpful here to give it more context